giza/agents/integrations/uniswap/nft_manager.py

```diff
@@ -0,0 +1,173 @@
+import os
+import time
+
+from ape import Contract
+
+from giza.agents.integrations.uniswap.constants import MAX_UINT_128
+from giza.agents.integrations.uniswap.utils import (
+    calc_amount0,
+    calc_amount1,
+    liquidity0,
+    liquidity1,
+    nearest_tick,
+    price_to_sqrtp,
+    price_to_tick,
+)
+
```

Comment:
Add logging

```
import logging

logger = logging.getLogger(__name__)
```

Add helpful logs for the user and also populate with `logger.debug` where it could be beneficial for the users